website/blog/2024-03-07-release-1.8.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Podman Desktop 1.8 Release
+description: Podman Desktop 1.8 has been released!
+slug: podman-desktop-release-1.8
+authors: deboer
+tags: [podman-desktop, release, kubernetes, openshift, onboarding, extensions, settings]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.8/selkie-family.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.8 Release! 🎉
+
+![Podman-desktop-1-8-hero](img/podman-desktop-release-1.8/selkie-family.png)
+
+We've got a new release with a ton of seal appeal! This release introduces:
+
+- **Podman 4.9.3**: [Podman 4.9.3](https://github.com/containers/podman/releases) is now included in both Windows and Mac installers.
+- **Kubernetes Explorer**: Get a sneak peek at the more advanced UI for working with Kubernetes clusters.
+- **Global Onboarding**: Configure and set up your environment without any hassle, with a set of guided flows.
+- **Learning Center**: Discover new use cases and capabilities for developers.
+- **Extension API Improvements**: Another big update to the extension API enabling more goodness for 🦭 Podman Desktop's extensions.
+- **Enhanced Builds, Pods List, and Troubleshooting Pages**: Build for different platforms, an upgraded pods view, and more.
+
+Podman Desktop 1.8 is now available. [Click here to download it](/downloads)!
+
+---
+
+## Release Details
+
+### Podman 4.9.3
+
+🦭 Podman 4.9.3 includes key fixes for stability and reliability issues reported by our users - especially
+if you are using a Mac M3. If you've been floundering we highly recommend updating!
+
+### Kubernetes Explorer
+
+Progressively introduced in past releases as experimental features, we're finally ready to expand
+our capabilities to help developers transition from containers to Kubernetes. In this release we
+are introducing a new set of capabilities that enable the developers to work with more Kubernetes
+resources, offering more granular and interactive control over your applications.
```

Comment:
```suggestion
resources, offering more granular and interactive control over your cluster.
```

maybe cluster better terminology? since we can view everything now (services, ingress, etc.)